apps/nuxt3-ssr/package.json

```diff
@@ -9,7 +9,6 @@
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
-    "postinstall": "nuxt prepare",
```

Comment:
by accident ?

Reply:
Yea, I need to remove that line in order to run emx2 locally. Forgot to reset the file.